app/helpers/generic_work_helper.rb

```diff
@@ -0,0 +1,11 @@
+module GenericWorkHelper
+
+  def render_collection_links(solr_doc)
+    return if solr_doc.collections.empty?
+    links = solr_doc.collections.map do |collection|
+      link_to collection.title_or_label, collections.collection_path(collection.id)
+    end
+    content_tag :span, ('Is part of: ' + links.join(', ')).html_safe
```

Comment:
@val99erie  If you have a chance to change the "Is part of:" to an I18N label that would be great!